src/Utils/getENV.tsx

```diff
@@ -2,6 +2,14 @@ import { data as sd } from "sharify"
 
 export function getENV(ENV_VAR) {
   const isServer = typeof window === "undefined"
-  const envVar = isServer ? process.env[ENV_VAR] : sd[ENV_VAR]
+
+  let envVar
+  if (isServer) {
+    const httpContext = require("express-http-context")
+    envVar = httpContext.get(ENV_VAR) ?? process.env[ENV_VAR]
```

Comment:
Uses new [nullish coalesing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) plugin, which always returns the left hand side of the operation unless `null` or `undefined`.

Reply:
Interesting. I didn't fully get immediately how different is than `||`. I do now recognize the subtle difference, but I think generally any sort of conditional assignment like this, if you are expecting a false-y/null/undefined to be possible (or even valid, esp. as a default), I almost always open a console to try it out (and/or mentally unpack it in my brain for a second).

Cool! I look forward to trying this.